DNB.js

```diff
@@ -0,0 +1,290 @@
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Ausgeplündert, zurückerstattet und entschädigt Arisierung und Wiedergutmachung in Mannheim",
+				"creators": [
+					{
+						"firstName": "Christiane",
+						"lastName": "Fritsche",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Ulrich",
+						"lastName": "Nieß",
+						"creatorType": "contributor"
+					},
+					{
+						"firstName": "Stadt",
+						"lastName": "Stadt Mannheim",
+						"creatorType": "contributor"
+					},
+					{
+						"firstName": "Johannes",
+						"lastName": "Paulmann",
+						"creatorType": "contributor"
+					},
+					{
+						"firstName": "Ulrich",
+						"lastName": "Nieß",
+						"creatorType": "contributor"
+					},
+					{
+						"firstName": "Susanne",
+						"lastName": "Schlösser",
+						"creatorType": "contributor"
+					}
+				],
+				"date": "2013",
+				"ISBN": "9783897357723",
+				"edition": "2. Aufl",
+				"libraryCatalog": "DNB",
+				"numPages": "960",
+				"place": "Ubstadt-Weiher Heidelberg Neustadt a.d.W. Basel",
+				"publisher": "Verl. Regionalkultur",
+				"series": "Sonderveröffentlichung des Stadtarchivs Mannheim - Institut für Stadtgeschichte",
+				"seriesNumber": "Bd. 39",
+				"attachments": [
+					{
+						"title": "Table of Contents PDF",
+						"mimeType": "application/pdf"
+					},
+					{
+						"title": "Abstract",
+						"mimeType": "text/html"
+					}
+				],
+				"tags": [
+					"(Produktform)Hardback",
+					"(Produktform)Hardback",
+					"(VLB-WN)1558: Hardcover, Softcover / Geschichte/Regionalgeschichte, Ländergeschichte",
+					"(VLB-WN)1558: Hardcover, Softcover / Geschichte/Regionalgeschichte, Ländergeschichte",
+					"Arisierung",
+					"Baden",
+					"Baden",
+					"Geschichte 1933-1969",
+					"Heimatgeschichte",
+					"Heimatgeschichte",
+					"Mannheim",
+					"Nachkriegszeit",
+					"Nachkriegszeit",
+					"Wiedergutmachung",
+					"Zweiter Weltkrieg",
+					"Zweiter Weltkrieg"
+				],
+				"notes": [
+					{
+						"note": "Literaturangaben"
+					}
+				],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "https://portal.dnb.de/opac.htm?method=showFullRecord&currentResultId=swiRef%3D953552055%26any&currentPosition=1",
+		"items": [
+			{
+				"itemType": "book",
+				"title": "Effiziente Entwurfsverfahren zur hardwarebasierten Signalverarbeitung elementarer Funktionen für die drahtlose Kommunikation / Jochen Rust",
+				"creators": [
+					{
+						"lastName": "Rust",
+						"creatorType": "author",
+						"firstName": "Jochen"
+					}
+				],
+				"date": "2014",
+				"ISBN": "978-3-8440-2706-8",
+				"libraryCatalog": "DNB",
+				"place": "Aachen",
+				"publisher": "Shaker",
+				"attachments": [
+					{},
+					{
+						"title": "Table of Contents PDF",
+						"mimeType": "application/pdf"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	}
+]
+/** END TEST CASES **/
```

Comment:
I have another test case where I need some help:

https://portal.dnb.de/opac.htm?method=simpleSearch&cqlMode=true&query=idn%3D1038274540

This produces an error but I don't understand what I am doing wrong. The error message is:

```
10:47:33 No MARC link found --> Use COinS translator
10:47:33 Translation using DNB failed: 
         string => TypeError: translate._sandboxManager.sandbox.Array is not a constructor
         stack => Zotero.Translate.Sandbox.Base.loadTranslator/safeTranslator.setHandler/<@chrome://zotero/content/xpcom/translation/translate.js:263:27
         Zotero.Translate.Base.prototype._runHandler@chrome://zotero/content/xpcom/translation/translate.js:989:20
         Zotero.Translate.Base.prototype._detectTranslatorsCollected@chrome://zotero/content/xpcom/translation/translate.js:1478:3
         Zotero.Translate.Base.prototype.complete@chrome://zotero/content/xpcom/translation/translate.js:1320:30
         Zotero.Translate.Search.prototype.complete@chrome://zotero/content/xpcom/translation/translate.js:2297:2
         Zotero.Translate.Base.prototype.decrementAsyncProcesses@chrome://zotero/content/xpcom/translation/translate.js:960:4
         Zotero.Translate.Base.prototype._detectTranslatorLoaded@chrome://zotero/content/xpcom/translation/translate.js:1469:3
         Zotero.Translate.Base.prototype._detect/<@chrome://zotero/content/xpcom/translation/translate.js:1450:17
         Zotero.Translate.Base.prototype._loadTranslator@chrome://zotero/content/xpcom/translation/translate.js:1512:16
         Zotero.Translate.Base.prototype._detect@chrome://zotero/content/xpcom/translation/translate.js:1449:1
         Zotero.Translate.Base.prototype.complete@chrome://zotero/content/xpcom/translation/translate.js:1317:5
         Zotero.Translate.Search.prototype.complete@chrome://zotero/content/xpcom/translation/translate.js:2297:2
         Zotero.Translate.Base.prototype.decrementAsyncProcesses@chrome://zotero/content/xpcom/translation/translate.js:960:4
         Zotero.Translate.Base.prototype._detectTranslatorLoaded@chrome://zotero/content/xpcom/translation/translate.js:1469:3
         Zotero.Translate.Base.prototype._detect/<@chrome://zotero/content/xpcom/translation/translate.js:1450:17
         Zotero.Translate.Base.prototype._loadTranslator@chrome://zotero/content/xpcom/translation/translate.js:1512:16
         Zotero.Translate.Base.prototype._detect@chrome://zotero/content/xpcom/translation/translate.js:1449:1
         Zotero.Translate.Base.prototype.complete@chrome://zotero/content/xpcom/translation/translate.js:1317:5
         Zotero.Translate.Search.prototype.complete@chrome://zotero/content/xpcom/translation/translate.js:2297:2
         Zotero.Translate.Base.prototype.decrementAsyncProcesses@chrome://zotero/content/xpcom/translation/translate.js:960:4
         Zotero.Translate.Base.prototype._detectTranslatorLoaded@chrome://zotero/content/xpcom/translation/translate.js:1469:3
         Zotero.Translate.Base.prototype._detect/<@chrome://zotero/content/xpcom/translation/translate.js:1450:17
         Zotero.Translate.Base.prototype._loadTranslator@chrome://zotero/content/xpcom/translation/translate.js:1512:16
         Zotero.Translate.Base.prototype._detect@chrome://zotero/content/xpcom/translation/translate.js:1449:1
         Zotero.Translate.Base.prototype.complete@chrome://zotero/content/xpcom/translation/translate.js:1317:5
         Zotero.Translate.Search.prototype.complete@chrome://zotero/content/xpcom/translation/translate.js:2297:2
         Zotero.Translate.Base.prototype.decrementAsyncProcesses@chrome://zotero/content/xpcom/translation/translate.js:960:4
         Zotero.Translate.Base.prototype._detectTranslatorLoaded@chrome://zotero/content/xpcom/translation/translate.js:1469:3
         Zotero.Translate.Base.prototype._detect/<@chrome://zotero/content/xpcom/translation/translate.js:1450:17
         Zotero.Translate.Base.prototype._loadTranslator@chrome://zotero/content/xpcom/translation/translate.js:1512:16
         Zotero.Translate.Base.prototype._detect@chrome://zotero/content/xpcom/translation/translate.js:1449:1
         Zotero.Translate.Base.prototype.complete@chrome://zotero/content/xpcom/translation/translate.js:1317:5
         Zotero.Translate.Search.prototype.complete@chrome://zotero/content/xpcom/translation/translate.js:2297:2
         Zotero.Translate.Base.prototype.decrementAsyncProcesses@chrome://zotero/content/xpcom/translation/translate.js:960:4
         Zotero.Translate.Base.prototype._detectTranslatorLoaded@chrome://zotero/content/xpcom/translation/translate.js:1469:3
         Zotero.Translate.Base.prototype._detect/<@chrome://zotero/content/xpcom/translation/translate.js:1450:17
         Zotero.Translate.Base.prototype._loadTranslator@chrome://zotero/content/xpcom/translation/translate.js:1512:16
         Zotero.Translate.Base.prototype._detect@chrome://zotero/content/xpcom/translation/translate.js:1449:1
         Zotero.Translate.Base.prototype.complete@chrome://zotero/content/xpcom/translation/translate.js:1317:5
         Zotero.Translate.Search.prototype.complete@chrome://zotero/content/xpcom/translation/translate.js:2297:2
         Zotero.Translate.Base.prototype.decrementAsyncProcesses@chrome://zotero/content/xpcom/translation/translate.js:960:4
         Zotero.Translate.Base.prototype._detectTranslatorLoaded@chrome://zotero/content/xpcom/translation/translate.js:1469:3
         Zotero.Translate.Base.prototype._detect/<@chrome://zotero/content/xpcom/translation/translate.js:1450:17
         Zotero.Translate.Base.prototype._loadTranslator@chrome://zotero/content/xpcom/translation/translate.js:1512:16
         Zotero.Translate.Base.prototype._detect@chrome://zotero/content/xpcom/translation/translate.js:1449:1
         Zotero.Translate.Base.prototype.complete@chrome://zotero/content/xpcom/translation/translate.js:1317:5
         Zotero.Translate.Search.prototype.complete@chrome://zotero/content/xpcom/translation/translate.js:2297:2
         Zotero.Translate.Base.prototype.decrementAsyncProcesses@chrome://zotero/content/xpcom/translation/translate.js:960:4
         Zotero.Translate.Base.prototype._detectTranslatorLoaded@chrome://zotero/content/xpcom/translation/translate.js:1469:3
         Zotero.Translate.Base.prototype._detect/<@chrome://zotero/content/xpcom/translation/translate.js:1450:17
         Zotero.Translate.Base.prototype._loadTranslator@chrome://zotero/content/xpcom/translation/translate.js:1512:16
         Zotero.Translate.Base.prototype._detect@chrome://zotero/content/xpcom/translation/translate.js:1449:1
         Zotero.Translate.Base.prototype.complete@chrome://zotero/content/xpcom/translation/translate.js:1317:5
         Zotero.Translate.Search.prototype.complete@chrome://zotero/content/xpcom/translation/translate.js:2297:2
         Zotero.Translate.Base.prototype.decrementAsyncProcesses@chrome://zotero/content/xpcom/translation/translate.js:960:4
         Zotero.Translate.Base.prototype._detectTranslatorLoaded@chrome://zotero/content/xpcom/translation/translate.js:1469:3
         Zotero.Translate.Base.prototype._detect/<@chrome://zotero/content/xpcom/translation/translate.js:1450:17
         Zotero.Translate.Base.prototype._loadTranslator@chrome://zotero/content/xpcom/translation/translate.js:1512:16
         Zotero.Translate.Base.prototype._detect@chrome://zotero/content/xpcom/translation/translate.js:1449:1
         Zotero.Translate.Base.prototype.complete@chrome://zotero/content/xpcom/translation/translate.js:1317:5
         Zotero.Translate.Search.prototype.complete@chrome://zotero/content/xpcom/translation/translate.js:2297:2
         Zotero.Translate.Base.prototype.decrementAsyncProcesses@chrome://zotero/content/xpcom/translation/translate.js:960:4
         Zotero.Translate.Base.prototype._detectTranslatorLoaded@chrome://zotero/content/xpcom/translation/translate.js:1469:3
         Zotero.Translate.Base.prototype._detect/<@chrome://zotero/content/xpcom/translation/translate.js:1450:17
         Zotero.Translate.Base.prototype._loadTranslator@chrome://zotero/content/xpcom/translation/translate.js:1512:16
         Zotero.Translate.Base.prototype._detect@chrome://zotero/content/xpcom/translation/translate.js:1449:1
         Zotero.Translate.Base.prototype.complete@chrome://zotero/content/xpcom/translation/translate.js:1317:5
         Zotero.Translate.Search.prototype.complete@chrome://zotero/content/xpcom/translation/translate.js:2297:2
         Zotero.Translate.Base.prototype.decrementAsyncProcesses@chrome://zotero/content/xpcom/translation/translate.js:960:4
         Zotero.Translate.Base.prototype._detectTranslatorLoaded@chrome://zotero/content/xpcom/translation/translate.js:1469:3
         Zotero.Translate.Base.prototype._detect/<@chrome://zotero/content/xpcom/translation/translate.js:1450:17
         Zotero.Translate.Base.prototype._loadTranslator@chrome://zotero/content/xpcom/translation/translate.js:1512:16
         Zotero.Translate.Base.prototype._detect@chrome://zotero/content/xpcom/translation/translate.js:1449:1
         Zotero.Translate.Base.prototype._getTranslatorsTranslatorsReceived@chrome://zotero/content/xpcom/translation/translate.js:1095:3
         Zotero.Translate.Base.prototype._getTranslatorsGetPotentialTranslators/<@chrome://zotero/content/xpcom/translation/translate.js:1062:28
         Zotero.Translators</this.getAllForType@chrome://zotero/content/xpcom/translation/translator.js:180:4
         Zotero.Translate.Base.prototype._getTranslatorsGetPotentialTranslators@chrome://zotero/content/xpcom/translation/translate.js:1061:1
         Zotero.Translate.Base.prototype.getTranslators@chrome://zotero/content/xpcom/translation/translate.js:1048:4
         Zotero.Translate.Search.prototype.getTranslators@chrome://zotero/content/xpcom/translation/translate.js:2254:9
         Zotero.Translate.Sandbox.Base.loadTranslator/safeTranslator.getTranslators@chrome://zotero/content/xpcom/translation/translate.js:302:12
         Zotero.Translate.Sandbox.Base.loadTranslator/safeTranslator[i]<@chrome://zotero/content/xpcom/translation/translate.js:401:9
         @chrome://zotero/content/xpcom/translation/translate_firefox.js:433:66
         completeItems@C:\Users\zumstein\AppData\Roaming\Mozilla\Firefox\Profiles\1imy4lix.default\zotero\translators\COinS.js:178:3
         completeCOinS@C:\Users\zumstein\AppData\Roaming\Mozilla\Firefox\Profiles\1imy4lix.default\zotero\translators\COinS.js:128:3
         doWeb@C:\Users\zumstein\AppData\Roaming\Mozilla\Firefox\Profiles\1imy4lix.default\zotero\translators\COinS.js:230:3
         Zotero.Translate.Base.prototype._translateTranslatorLoaded@chrome://zotero/content/xpcom/translation/translate.js:1189:4
         Zotero.Translate.Web.prototype._translateTranslatorLoaded@chrome://zotero/content/xpcom/translation/translate.js:1770:3
         Zotero.Translate.Base.prototype.translate/</<@chrome://zotero/content/xpcom/translation/translate.js:1162:51
         Zotero.Translate.Base.prototype._loadTranslator@chrome://zotero/content/xpcom/translation/translate.js:1512:16
         Zotero.Translate.Base.prototype.translate/<@chrome://zotero/content/xpcom/translation/translate.js:1162:7
         Zotero.Translators</this.get@chrome://zotero/content/xpcom/translation/translator.js:162:4
         Zotero.Translate.Base.prototype.translate@chrome://zotero/content/xpcom/translation/translate.js:1159:1
         Zotero.Translate.Web.prototype.translate@chrome://zotero/content/xpcom/translation/translate.js:1761:2
         Zotero.Translate.Sandbox.Base.loadTranslator/safeTranslator.translate@chrome://zotero/content/xpcom/translation/translate.js:317:12
         Zotero.Translate.Sandbox.Base.loadTranslator/safeTranslator[i]<@chrome://zotero/content/xpcom/translation/translate.js:401:9
         @chrome://zotero/content/xpcom/translation/translate_firefox.js:433:66
         scrape@DNB:132:3
         doWeb@DNB:96:3
         Zotero.Translate.Base.prototype._translateTranslatorLoaded@chrome://zotero/content/xpcom/translation/translate.js:1189:4
         Zotero.Translate.Web.prototype._translateTranslatorLoaded@chrome://zotero/content/xpcom/translation/translate.js:1770:3
         Zotero.Translate.Base.prototype.translate/<@chrome://zotero/content/xpcom/translation/translate.js:1156:58
         Zotero.Translate.Base.prototype._loadTranslator@chrome://zotero/content/xpcom/translation/translate.js:1512:16
         Zotero.Translate.Base.prototype.translate@chrome://zotero/content/xpcom/translation/translate.js:1156:4
         Zotero.Translate.Web.prototype.translate@chrome://zotero/content/xpcom/translation/translate.js:1761:2
         _run@chrome://scaffold/content/scaffold.js:408:4
         run@chrome://scaffold/content/scaffold.js:352:4
         oncommand@chrome://scaffold/content/scaffold.xul:1:1
         
         url => https://portal.dnb.de/opac.htm?method=showFullRecord&currentResultId=tit+all+%22seattle%22+and+jhr+within+%22*+2013%22+and+jhr+within+%222000+*%22%26any&currentPosition=4
         downloadAssociatedFiles => true
         automaticSnapshots => true
```

Any help?